src/rev_ai/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """Top-level package for rev_ai"""
 
-__version__ = '2.10.0'
+__version__ = '2.10.1'
```

Comment:
@k-weng looking at [semver spec](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api) I am thinking of bumping the PATCH version since we are not adding functionality but are introducing a change.